lighthouse-core/lib/i18n/locales/ar.json

```diff
@@ -875,6 +875,24 @@
   "lighthouse-core/config/default-config.js | seoCategoryTitle": {
     "message": "تحسين محركات البحث"
   },
+  "lighthouse-core/config/default-config.js | seoContentGroupDescription": {
+    "message": "Format your HTML in a way that enables crawlers to better understand your app’s content."
```

Comment:
they just didn't get to all of them?

Reply:
oh I guess this is what "will keep this updated as new translations come in." was about :)

Reply:
Yeah, some languages aren't in yet :wink: , this is just a WIP as they come in.